src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/SurveySvcRestClientImpl.java

```diff
@@ -123,46 +124,55 @@ private SurveyDTO getSurveyDtoFromResponseEntity(ResponseEntity<String> response
   }
 
   @Override
-  public SurveyDTO findSurvey(UUID surveyId) throws RestClientException {
+  public SurveyDTO findSurvey(final UUID surveyId) throws CTPException {
     UriComponents uriComponents =
         restUtility.createUriComponents(
             appConfig.getSurveySvc().getSurveyDetailPath(), null, surveyId);
 
     HttpEntity<?> httpEntity = restUtility.createHttpEntity(null);
+    ResponseEntity<String> responseEntity = null;
 
     try {
       log.debug(
           "about to get to the Survey SVC with surveyId {} from {}",
           surveyId,
           uriComponents.toUri());
-      ResponseEntity<String> responseEntity =
+      responseEntity =
           restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
-
       return getSurveyDtoFromResponseEntity(responseEntity);
     } catch (RestClientException e) {
-      return null;
+      if (responseEntity != null && responseEntity.getStatusCode().is4xxClientError()) {
+        throw new CTPException(
+            CTPException.Fault.BAD_REQUEST,
+            String.format("There was a 4xx error: %s", e.toString()));
+      }
+      throw new CTPException(
+          CTPException.Fault.SYSTEM_ERROR, String.format("System error: %s", e.toString()));
     }
   }
 
   @Override
-  public SurveyDTO findSurveyByRef(String surveyRef) throws RestClientException {
+  public SurveyDTO findSurveyByRef(final String surveyRef) throws CTPException {
     UriComponents uriComponents =
         restUtility.createUriComponents(
             appConfig.getSurveySvc().getSurveyRefPath(), null, surveyRef);
 
     HttpEntity<?> httpEntity = restUtility.createHttpEntity(null);
+    ResponseEntity<String> responseEntity = null;
 
     try {
-      log.debug(
-          "about to get to the Survey SVC with surveyRef {} from {}",
-          surveyRef,
-          uriComponents.toUri());
-      ResponseEntity<String> responseEntity =
+      responseEntity =
           restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
 
       return getSurveyDtoFromResponseEntity(responseEntity);
     } catch (RestClientException e) {
-      return null;
+      if (responseEntity != null && responseEntity.getStatusCode().is4xxClientError()) {
```

Comment:
Not sure if this is adding any value. The behaviour spring mvc has is that it will already do this.